natss/cmd/channel_dispatcher/main.go

```diff
@@ -99,8 +101,9 @@ func main() {
 
 	// Watch the logging config map and dynamically update logging levels.
 	opt.ConfigMapWatcher.Watch(logging.ConfigMapName(), logging.UpdateLevelFromConfigMap(logger, atomicLevel, logconfig.Controller))
-	// TODO: Watch the observability config map and dynamically update metrics exporter.
-	//opt.ConfigMapWatcher.Watch(metrics.ObservabilityConfigName, metrics.UpdateExporterFromConfigMap(component, logger))
+
+	// Watch the observability config map
+	opt.ConfigMapWatcher.Watch(metrics.ConfigMapName(), metrics.UpdateExporterFromConfigMap(component, logger))
```

Comment:
the dispatchers currently do not report stuff. But I think we can add this for now, and then see what metrics we want for all channel dispatchers...

Reply:
my bad, reporting at least what the pkg controller stats_reporter does...